tsconfig.json

```
@@ -3,6 +3,7 @@
     "target": "es5",
     "module": "commonjs",
```

```suggestion
    "module": "es2015",
```
The microbundle `commonjs` error was actually rollup error. And these turn out to be correct. bili doesn't show error because it simply ignores this field!

---

I've switched to `es2015` module as requested but I'm back to having the issue where the tests won't run due to the `ava` module not being found (and also `src` files interestingly). I'm not super familiar with the various typescript module styles and how they differ but do you know what might be causing this?

---

Ah, i see so just keep it `commonjs`

---

Done - although looks like `allowSyntheticDefaultImports: true` could potentially fix the issue?

https://github.com/Microsoft/TypeScript/issues/3337#issuecomment-307594895